packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -377,6 +383,8 @@ function _Autocomplete<ItemType>(
                 />
               )}
           </Popover.Content>
+        ) : (
+          <div {...menuProps} className={cx(styles.hidden)} />
```

Comment:
Could you add a comment here explaining why we need it, so we don't remove it later?